### ISSUE TITLE
Grandchildren are Inbreeding

### ISSUE THREAD

**Issue by GameRKade** [WildBamaBoy/minecraft-comes-alive#830]
- [x] I am running the latest mod versions of MCA and RadixCore.
- [x] I can reproduce this issue with just MCA and RadixCore installed.
- [x] I can provide my Minecraft version and MCA version.
- [x] I have searched for the issue previously and it was either not previously reported, or previously fixed and I'm having the same problem.
- [] I am crashing and can provide my crash report.

## Versions
Minecraft 1.10.2
MCA 5.2.3

## Issue Description
In my Minecraft survival, I had several children. One of them happened to get married and have a child, giving me a grandchild. However, the game didn't recognize she was my granddaughter, so she treated me like a stranger. I didn't find this too much of an issue, until what happened next.

One day, I approach my granddaughter and see that she's holding a baby. I was happy to have a great granddaughter, but I was curious as to who she married. I right-clicked her and checked her marriage info, and it said that she had married my son.

Basically, my granddaughter had just married her uncle, and now I have a half-granddaughter half-great-granddaughter.

As well as that, if you have a grandchild and divorce your spouse, you are able to marry and have children with your grandchild.

## Reproduce Steps
(1) Have a child, and let them grow up to adult
(2) Make your child get married and have a grandchild
(3) Wait for your grandchild to grow up
(4) Your grandchild can marry any of your children, grandchildren, any great grandchild, your spouse, or even you yourself.

**Comment by WildBamaBoy** [comment]
There's not much I can do about this.
